cmd/agent/app/run.go

```diff
@@ -409,6 +410,14 @@ func StartAgent() error {
 		}
 	}
 
+	// Start NetFlow server
```

Comment:
This should happen later, after `LoadComponents` initializes the tagger, which is used by demux. Starting the server here leaves a possibility for a race between demux handling the traffic and the tagger initialization.